### ISSUE TITLE
Add NATS support

### ISSUE THREAD

**Issue by cpitkin** [hashicorp/faas-nomad#31]
Add NATS support to allow for async functions.

**Comment by nicholasjackson** [comment]
@cpitkin I have a branch now /nats which adds configurable options for function timeouts which allows values greater than the default of 30s.  This is nearly done will hopefully merge today sometime.

**Comment by nicholasjackson** [comment]
@cpitkin this has now been merged to master the new image version is v0.2.23 which accepts a new parameter set:

```hcl
 args = [
   "-nomad_region", "${NOMAD_REGION}",
   "-nomad_addr", "${NOMAD_IP_http}:4646",
   "-consul_addr", "${NOMAD_IP_http}:8500",
   "-statsd_addr", "${NOMAD_ADDR_statsd_statsd}",
   "-node_addr", "${NOMAD_IP_http}",
   "-logger_format", "json",
   "-logger_output", "/logs/nomadd.log"
   "-function_timeout", "5m"
]
```

You should now be able to set the function timeout as requested.
